lib/potluck.rb

```diff
@@ -0,0 +1,34 @@
+require './lib/dish.rb'
+
+class Potluck
+  attr_reader :date,
+              :dishes
+
+  def initialize(date)
+    @date = date
+    @dishes = []
+    @category_array = []
+    #couscous_salad = Dish.new("Couscous Salad",:appetizer)
+    #cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+
+  end
+
+  def add_dish(dish)
+    dishes << dish
+    return dish
+  end
+
+  def get_all_from_category(category)
```

Comment:
This logic looks REALLY great!  I think there is a bit of room to refactor so that you are not using an instance variable to store the category array, but overall really great job!